packages/components/src/components/chat/ContextContainer.vue

```diff
@@ -4,11 +4,12 @@
     data-cy="context-container"
     :data-handle="valueHandle"
     :data-reference="isReference"
+    :title="location"
   >
     <div
       :class="{
         'context-container__header': 1,
-        'context-container__header--collapsable': isCollapsable,
+        'context-container__header--collapsable': isCollapsable || !isAppliable,
```

Comment:
There's a 1px border on the bottom of this header that shouldn't be present

Reply:
Should never be present, or shouldn't be present for items that aren't appliable (e.g. files)?